data.access/src/test/java/data/access/DatabaseUtility.java

```diff
@@ -0,0 +1,139 @@
+package data.access;
+import java.sql.*;
+import java.util.ArrayList;
+
+
+public class DatabaseUtility implements DatabaseAccessor {
+
+	static final String DB_URL = "jdbc:mysql://localhost:3306/sakila";
```

Comment:
Make the utility more general by setting conn string and credentials outside of the class.